src/entities.rs

```diff
@@ -521,6 +524,49 @@ impl Entities {
         }
     }
 
+    fn extend_meta(&mut self, new_len: usize, include_last: bool) {
+        let upper_limit = if include_last {
+            new_len as u32
+        } else {
+            new_len as u32 - 1
+        };
+
+        self.pending.extend((self.meta.len() as u32)..upper_limit);
+        let new_free_cursor = self.pending.len() as isize;
+        *self.free_cursor.get_mut() = new_free_cursor;
+        self.meta.resize(new_len, EntityMeta::EMPTY);
+    }
+
+    pub fn push_generations(&mut self, generations: &[u32]) {
+        if self.meta.len() < generations.len() {
+            self.extend_meta(generations.len(), true);
+        }
+
+        for (index, meta) in self.meta.iter_mut().enumerate() {
+            meta.generation = NonZeroU32::new(generations[index]).expect("Could not create non zero u32");
```

Comment:
This panic message is not helpful. Either write a message that gives a developer seeing it useful context, or use a concise `unwrap`. Panic messages should contain information that isn't already immediately obvious from the backtrace.

Reply:
Replaced with unwrap.